app/components/spina/forms/search_component.html.erb

```diff
@@ -1,16 +1,12 @@
 <div class="flex items-center relative">
-  <div class="absolute left-3" style="left: 0.75rem;">
+  <div class="absolute left-2">
     <%= helpers.heroicon('search', style: :solid, class: 'text-gray-300 w-5 h-5') %>
   </div>
 
-  <%= @f.search_field @method, 
+  <%= f.search_field @method, 
     value: params[@method.to_sym],
-    class: "form-input font-medium block pl-10 pr-2 py-[0.3rem]"
+    placeholder: t("spina.search"),
+    class: "form-input rounded-full pl-8 text-sm h-9 w-full",
+    data: {action: "input->form#requestSubmit"}
```

Comment:
The search on input is a nice touch. 👌

Reply:
I was working on another search input for the new PageLink part which includes a debounce method. When that's merged, this will also debounce so it's not firing search requests like crazy as it does now.